src/EventListener/DoctrineTaskExecutionListener.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of php-task library.
+ *
+ * (c) php-task
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Task\TaskBundle\EventListener;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Task\Event\TaskExecutionEvent;
+
+/**
+ * Listens on task-execution events.
+ */
+class DoctrineTaskExecutionListener
```

Comment:
also like this we could handle everything for doctrine - i dont see the difference